src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -71,6 +71,7 @@ export class CourseUpdateComponent implements OnInit {
     faExclamationTriangle = faExclamationTriangle;
     faPen = faPen;
 
+    faqEnabled = true; //default value
```

Comment:
I think we can omit this comment. The other booleans don't do this, either

```suggestion
    faqEnabled = true;
```

Reply:
some do it, some do not. In the file, 10 lines above some have to comment. But i will simply remove all of them, should be intuitive